flutter/news/test/features/frontpage/presentation/bloc/articles_cubit_test.dart

```diff
@@ -56,10 +56,7 @@ void main() {
       return ArticlesCubit(repository: repository);
     },
     act: (cubit) => cubit.getTopHeadlines(),
-    expect: () => <ArticlesState>[
-      const ArticlesState.loading(),
-      const ArticlesState.error(),
-    ],
+    expect: () => [isA<Loading>(), isA<Error>()],
```

Comment:
Why this change? I think previously was good enough, also it removes the need for making public `_Loading`, `_Error`, etc

Reply:
I changed here because when i added a message to the error i had to had the matching message here, and i think we only need to make sure the state is of the correct Type